src/main/resources/modules/EXM125-8.0.000-r4.json

```diff
@@ -139,18 +196,50 @@
         {
           "transition": "Set_Denominator",
           "condition": {
-            "condition_type": "And",
+            "condition_type": "Or",
             "conditions": [
               {
-                "condition_type": "Attribute",
-                "attribute": "Encounter Occurrence",
-                "operator": "==",
-                "value": true
+                "condition_type": "And",
+                "conditions": [
+                  {
+                    "condition_type": "Attribute",
+                    "attribute": "Encounter Occurrence",
+                    "operator": "==",
+                    "value": true
+                  },
+                  {
+                    "condition_type": "Attribute",
+                    "attribute": "Mammography Occurrence",
+                    "operator": "is nil"
+                  }
+                ]
               },
               {
-                "condition_type": "Attribute",
-                "attribute": "Mammography Occurrence",
-                "operator": "is nil"
+                "condition_type": "And",
+                "conditions": [
+                  {
+                    "condition_type": "And",
+                    "conditions": [
+                      {
+                        "condition_type": "Attribute",
+                        "attribute": "Encounter Occurrence",
+                        "operator": "==",
+                        "value": true
+                      },
+                      {
+                        "condition_type": "Attribute",
+                        "attribute": "Mammography Occurrence",
+                        "operator": "==",
+                        "value": true
+                      }
+                    ]
+                  },
+                  {
+                    "condition_type": "Attribute",
+                    "attribute": "In Measurement Period",
```

Comment:
`In Measurement Period` needs to be true for any patient in the IPP since this is actually set for the initial qualifying encounter. With the way that this module is set up, the mammography always happens right after the initial qualifying encounter, so that's why we don't have to check the timing of the mammography occurence.

Reply:
I talked with Matt and we have the JSON logic in the module that checks whether the date comes after the start of the measurement period. He made it sound like we do not need to add any guards to this module. Does that mean this module is good in terms of measurement period?

Reply:
Maybe we can add another task to eventually switch the measurement period logic for all the modules to the logic that appears in this module's JSON file